client/src/core/RuntimeTypes.res

```diff
@@ -122,6 +126,7 @@ module FQFnName = {
   }
 }
 
+// WHAT consider renaming to MatchPattern, as opposed to an upcoming LetPattern/DeconstructPattern
```

Comment:
I don't think comments like this are super helpful

Reply:
This was more me asking a question indirectly:
We don't yet have a mechanism for deconstructing in a let, and I was playing with names briefly, to later ask if you had any preference that I can keep in mind.